src/Behat/Behat/Context/Cli/InteractiveContextIdentifier.php

```diff
@@ -69,7 +69,7 @@ public function guessTargetContextClass(ContextEnvironment $environment)
         }
 
         $message = $this->translator->trans('snippet_context_choice', array('%count%' => $suiteName), 'output');
-        $choices = array_values(array_merge(array('None'), $contextClasses));
+        $choices = array_merge(array('None'), $contextClasses);
```

Comment:
The merged array is already an integers indexed list, `array_values()` does not do anything here